index.html

```diff
@@ -0,0 +1,213 @@
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link href=”output.css” rel=”stylesheet”>
+        <link href=”main.css” rel=”stylesheet”>
+        <link rel="stylesheet" href="style.css"/>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Open+Sans+Condensed:wght@700&display=swap" rel="stylesheet">
+        <title>Portfolio of Sami</title>
+            <link
+              rel="icon"
+              type="image/png"
+              sizes="32x32"
+              href="/favicon-32x32.png"
+            />
+            <link
+              rel="icon"
+              type="image/png"
+              sizes="16x16"
+              href="/favicon-16x16.png"
+            />
+    
+            <link
+              rel="stylesheet"
+              href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
+              integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w=="
+              crossorigin="anonymous"
+              referrerpolicy="no-referrer"
+            />
+          
+    </head>
+    
+    <body class="">
+      <section  class= "container  flex flex-row  ">
+            <div class="mx-12 mt-12 mb-12 ">
+                <p class=" text-4xl font-Sans-Condensed text-gray-800 font-bold ">Hi, I'm Sami Zad</p>
+                <p class="pl-20 text-2xl text-gray-700 font-light">I'm a Front-End Developer</p>
+                <p class=" text-2xl text-yellow-700 font-light">I'm a Hardworking and reliable Front-End developer
+                focused on going above
+                </p>
+            </div>
+            <div class= " mx-12 mt-32 mb-12">
+                <button class="bg-gray-200 hover:bg-yellow-700 text-gray-700 font-semibold hover:text-white py-2 px-4 rounded-3xl">
+                    Download CV
+                  </button>
+            </div>
+            <!---
+           <div>
+            <img src="./Images/myimage.jpg" alt="Samizad" class="shadow rounded max-w-full h-auto align-middle border-none" />
+           </div>-->
```

Comment:
It's not good practice to have code commented out in your files. Unless they are a note for the future... but in this case, you should leave a message before it (comment) saying why you left it like this.